css/media.css

```diff
@@ -0,0 +1,319 @@
+@media(min-width: 320px) {
+    body {
+        box-sizing: border-box;
+        font-family: Montserrat, sans-serif;
+        background-color: #fff;
+    }
+    .header {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla_car/pullRequest?prid=5929182)